src/api/app/models/binary_release.rb

```diff
@@ -184,6 +182,12 @@ def reset_cache
     Rails.cache.delete("xml_binary_release_#{cache_key}")
   end
 
+  def indentical_to?(binary_hash)
+    binary_disturl == binary_hash['disturl'] &&
+      binary_supportstatus == binary_hash['supportstatus'] &&
+      binary_buildtime.present? &&
```

Comment:
`present?` is not needed, `build_buildtime` is either a time or `nil`, so you can remove the `present` as it is in my [previous comment](https://github.com/openSUSE/open-build-service/pull/5149#discussion_r196119971) :wink: